src/app/shell/details/details-tabs/reviews/reviews.component.ts

```diff
@@ -76,6 +81,8 @@ export class ReviewsComponent implements OnInit, OnDestroy {
       this.parent = parent;
       //this.store.dispatch(new GetApplicationsByParentId(parent.id)); // TODO: check if parent has applciation
```

Comment:
remove this commented line, it is not needed anymore
add here     this.checkIfAllowedToReview(this.parent.id);
    this.checkIfReviewed(this.parent.id);
because only in subscription we can be sure that parent is not undefined